applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -325,8 +326,11 @@ static void cmd_send(struct k_work *work)
 
 	ARG_UNUSED(work);
 
-	/* Make sure the string is 0-terminated */
-	at_buf[MIN(at_buf_len, AT_MAX_CMD_LEN - 1)] = 0;
+	if (at_buf_overflow) {
+		rsp_send(OVERFLOW_STR, sizeof(OVERFLOW_STR) - 1);
+		rsp_send(ERROR_STR, sizeof(ERROR_STR) - 1);
```

Comment:
Do we need 2 Error string here? OVERFLOW_STR and ERROR_STR.

Comparing to the case FATAL_STR, there is only FATAL_STR sent.

Reply:
As we talked on Teams, overflow string is URC and not valid [AT response](https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/at_syntax_response.html?cp=2_1_1_3)

Reply:
I see.